### DISCUSSION TITLE
essay/random

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#13] [Announcements]
# essay/random

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/essay/random

**Comment by sata0109** [comment]
讽刺一些人（总不能是讽刺女性😅）不懂得专业知识而且蛮横火爆？？？？

**Comment by Eliza-Liang** [comment]
偶然发现的网站

**Comment by wurencaideli** [comment]
短文推荐
[鲜花盛开的古镇](https://www.dumogu.top/essay/info/hQbzbP7Bt7mJ5naOcexRh)

**Comment by Eliza-Liang** [comment]
Dear my friends,I have received your email. I will reply it as soon as possible.llq
